numpy/f2py/crackfortran.py

```diff
@@ -696,7 +696,8 @@ def _simplifyargs(argsline):
     return ','.join(a)
 
 crackline_re_1 = re.compile(r'\s*(?P<result>\b[a-z]+\w*\b)\s*=.*', re.I)
-
+crackline_bind_1 = re.compile(r'\s*(?P<bind>\b[a-z]+\w*\b)\s*=.*', re.I)
+crackline_bindlang = re.compile(r'\s*bind\(\s*(?P<lang>[^,]+)\s*,\s*name\s*=\s*"(?P<lang_name>[^"]+)"\s*\)', re.I)
```

Comment:
These long templates are pretty much unreadable. Might be nice to use a generated parser at some point.

Reply:
Yeah they take forever to write too 😅, would be nice to explore Lark at some time. The problem is most parsers don't actually support regex based parsing (which is needed for fortran because of the whitespace stuff).